src/model/transaction/AggregateTransaction.ts

```diff
@@ -372,11 +372,23 @@ export class AggregateTransaction extends Transaction {
      * Generate inner transaction root hash (merkle tree)
      * @returns {Uint8Array}
      */
-    private calculateInnerTransactionHash(): Uint8Array {
-        const builder = new MerkleHashBuilder(SHA3Hasher.createHasher);
+    private calculateInnerTransactionHash(networkType: NetworkType): Uint8Array {
```

Comment:
Seems networkType is not used at all. should it be removed?

Reply:
that one can be removed